src/api/posts/src/storage.js

```diff
@@ -22,6 +22,26 @@ const createInvalidFeedKey = (id) => createFeedKey(id).concat(invalidSuffix);
 // "NirlSYranl" to "t:feed:NirlSYranl:delayed"
 const createDelayedFeedKey = (id) => createFeedKey(id).concat(delayedSuffix);
 
+const getInvalidKeys = async () => {
+  let keysArray = [];
+  const stream = redis.scanStream({
+    match: 't:feed:*:invalid',
+  });
+  return new Promise((res, rej) => {
+    stream.on('data', async (keys) => {
```

Comment:
You don't need `async` here, since you aren't `await`ing any promise.

Reply:
I would also set a default for `keys` so you don't have to check its value: `(keys = []) => {...}`